<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QGroupBox" name="groupBox_9">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="NON_power">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Arial Black</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Power</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QDoubleSpinBox" name="SPB_power">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>80</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="minimum">
                 <double>-80.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.010000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="CBOX_power">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>70</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="currentIndex">
                 <number>1</number>
                </property>
                <item>
                 <property name="text">
                  <string>Vpp</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>dBm</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>mW</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="SPB_power_step">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>80</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="minimum">
                 <double>0.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>999.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.010000000000000</double>
                </property>
                <property name="value">
                 <double>0.010000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QSlider" name="SLB_power">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksBothSides</enum>
              </property>
              <property name="tickInterval">
               <number>5</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_10">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="NON_freq">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Arial Black</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Frequency</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <widget class="QDoubleSpinBox" name="SPB_freq">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>80</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>1000000000000000000.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="CBOX_freq">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>70</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="currentIndex">
                 <number>1</number>
                </property>
                <item>
                 <property name="text">
                  <string>kHz</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>MHz</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>GHz</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="SPB_freq_step">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>80</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="minimum">
                 <double>0.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>999.000000000000000</double>
                </property>
                <property name="value">
                 <double>1.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QSlider" name="SLB_freq">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximum">
               <number>20</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksBothSides</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_11">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="NON_phase">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Arial Black</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Phase</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_13">
              <item>
               <widget class="QDoubleSpinBox" name="SPB_phase">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>80</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="minimum">
                 <double>-359.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>360.000000000000000</double>
                </property>
                <property name="value">
                 <double>0.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="CBOX_phase">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>70</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>16777215</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>degree</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>radian</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="SPB_phase_step">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>80</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="minimum">
                 <double>0.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>999.000000000000000</double>
                </property>
                <property name="value">
                 <double>1.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QSlider" name="SLB_phase">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="value">
               <number>49</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksBothSides</enum>
              </property>
              <property name="tickInterval">
               <number>5</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QComboBox" name="CBOX_channel">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSpinBox" name="SPB_ref">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="CBOX_ref">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>MHz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>GHz</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="RDO_Internal">
          <property name="text">
           <string>Internal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="RDO_external">
          <property name="text">
           <string>External</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BTN_lock">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial Black</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Lock</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="BTN_oscillo">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../wave.png</normaloff>../wave.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BTN_connect">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BTN_on">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>On</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>BTN_on</sender>
   <signal>clicked(bool)</signal>
   <receiver>Form</receiver>
   <slot>pressedPowerButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>637</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>967</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SPB_power</sender>
   <signal>editingFinished()</signal>
   <receiver>Form</receiver>
   <slot>editedPower()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>1152</x>
     <y>-35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SPB_freq</sender>
   <signal>editingFinished()</signal>
   <receiver>Form</receiver>
   <slot>editedFreq()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>1080</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SPB_phase</sender>
   <signal>editingFinished()</signal>
   <receiver>Form</receiver>
   <slot>editedPhase()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>857</x>
     <y>-51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SPB_power_step</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Form</receiver>
   <slot>changedPowerStep()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>892</x>
     <y>-127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SPB_freq_step</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Form</receiver>
   <slot>changedFreqStep()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>925</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SPB_phase_step</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Form</receiver>
   <slot>changedPhaseStep()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>840</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BTN_connect</sender>
   <signal>clicked(bool)</signal>
   <receiver>Form</receiver>
   <slot>pressedConnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>1153</x>
     <y>600</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BTN_lock</sender>
   <signal>clicked(bool)</signal>
   <receiver>Form</receiver>
   <slot>pressedLockButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>587</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>941</x>
     <y>533</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changedFreqUnit()</slot>
  <slot>changedPowerUnit()</slot>
  <slot>changedPhaseUnit()</slot>
  <slot>changedPower()</slot>
  <slot>changedFreq()</slot>
  <slot>changedPhase()</slot>
  <slot>editedPower()</slot>
  <slot>editedFreq()</slot>
  <slot>editedPhase()</slot>
  <slot>changedPowerStep()</slot>
  <slot>changedFreqStep()</slot>
  <slot>changedPhaseStep()</slot>
  <slot>pressedPowerButton()</slot>
  <slot>pressedConnect()</slot>
  <slot>pressedLockButton()</slot>
 </slots>
</ui>
